### PR TITLE
add glide, travis

### PR DESCRIPTION
**Pull request by smola**

### CODE REVIEW

**Comment by bzz** [comment]
🎉  for discarding `srcd.works` in favor of `gopkg.in`

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+go:
+  - 1.8
+  - tip
+
+matrix:
+  fast_finish: true
+  allow_failures:
+    - go: tip
+
+sudo: required
```
Comment (bzz) [inline .travis.yml]: This line prevents us from using container-based builds which tend to be much faster.  Is there a reason for it?

Why not try to run it on [modern infrastructure](https://blog.travis-ci.com/2017-06-19-trusty-updates-2017-Q2) from scratch by replacing this with
```
sudo: false
dist: trusty
```

**Comment by smola** [comment]
Depends on https://github.com/src-d/go-git/pull/453

**Comment by bzz** [comment]
Looks great @smola !

Quick question - in this case, do we wait for 
 - src-d/go-git#453
 - new go-git.v4 release

before merging this one? Or can glide be used to pin-point go-git version \wo a need for release?

**Comment by smola** [comment]
@bzz we can point to a commit once its merged.

**Comment by alcortesm** [comment]
> alcortesm approved these changes 15 hours ago

**Pull request merged**